modules/users/client/controllers/settings/manage-social-accounts.client.controller.js

```diff
@@ -42,5 +43,15 @@
     function onRemoveSocialAccountError(response) {
       Notification.error({ message: response.message, title: '<i class="glyphicon glyphicon-remove"></i> Remove failed!' });
     }
+
+    // OAuth provider request
+    function callOauthProvider(url) {
+      if ($state.previous && $state.previous.href) {
+        url += '?redirect_to=' + encodeURIComponent($state.$current.url.prefix);
```

Comment:
`$state.$current.url.prefix` contains /settings/accounts, is that your question?